TheSuperMighty/MightyIO-iOS-Pod
# Features/415/sample app

closes #415
## To Test:
1. Launch app on iphone (IAP won't work on simulator).
2. Click "Get Mighty Items" and an alert view should trigger listing Mighty Items in the logged in account.
3. Click "Process Transaction" and a transaction should begin.  Log in with gd@supermighty.com : Scr@pp1e to complete.
4. Check Parse for record of purchase.
5. Click Process transaction with block.  Modal should trigger when Super Mighty has processed transaction.  Check Parse for record of transaction.
6. Click Open Facebook Share Modal With Custom Text.  A share modal should init with the text "I just made a Mighty Purchase"
7. Click Open Facebook Share Modal.  A share modal with the text "Test share text." should appear.  Verify this is the text listed in settings table in the Parse DB
